app/main.py

```diff
@@ -1 +1,5 @@
-# write your code here
+def copy_file(command: str) -> None:
+    sort_command = command.split()
+    with (open(sort_command[1], "r") as file_in,
+          open(sort_command[2], "w") as file_out):
+        file_out.write(f"{file_in}")
```

Comment:
Instead of f-string use .`read()` method
```suggestion
        file_out.write(file_in.read())
```